staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -418,6 +421,15 @@ func (s *GenericAPIServer) installAPIResources(apiPrefix string, apiGroupInfo *A
 			apiGroupVersion.OptionsExternalVersion = apiGroupInfo.OptionsExternalVersion
 		}
 		apiGroupVersion.OpenAPIModels = openAPIModels
+
+		if openAPIModels != nil && utilfeature.DefaultFeatureGate.Enabled(features.ServerSideApply) {
+			typeConverter, err := fieldmanager.NewTypeConverter(openAPIModels, false)
+			if err != nil {
+				return err
```

Comment:
This is probably not ideal, we should probably silently fail. Should we still record the error somewhere?

Reply:
So we are failing somewhere because of that. If this is about a built-in type, we should clearly not fail silently.

Reply:
ack, fixed.